website/docs/cloud-docs/api-docs/data-retention-policies.mdx

```diff
@@ -0,0 +1,301 @@
+| [404][] | [JSON API error object][] | Target resource not found, or user unauthorized to perform action. |
+
+### Request Body
+
+This `POST` endpoint requires a JSON object with the following properties as a request payload.
+
+| Key path          | Type   | Default | Description                    |
+| ----------------- | ------ | ------- | ------------------------------ |
+| `data.type`       | string |         | A [data retention policy type](#types-of-data-retention-policies) |
+| `data.attributes` | object |         | The attributes for the [chosen policy type](#types-of-data-retention-policies) |
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "deleteOlderThanNDays": 33
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/workspaces/ws-UYv6RYM8fVhzeGG5/relationships/data-retention-policy
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "drp-Kh74zpKVGNWbpugQ",
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "delete-older-than-n-days": 33
+    },
+    "relationships": {
+      "target": {
+        "data": {
+          "id": "ws-7aiqKYf6ejMFdtWS",
+          "type": "workspaces"
+        }
+      }
+    }
+  }
+}
+```
+
+## Remove Data Retention Policy
+
+This endpoint removes the data retention policy explicitly set on a target resource.
+
+| Resource      | API Endpoint                                    | 
+| --------------| ----------------------------------------------- |
+| [Workspaces](/terraform/cloud-docs/api-docs/workspaces#remove-data-retention-policy)       | `DELETE /workspaces/:workspace_id/relationships/data-retention-policy`         |
+| [Organizations](/terraform/cloud-docs/api-docs/organizations#remove-data-retention-policy) | `DELETE /organizations/:organization_name/relationships/data-retention-policy` |
+
+| Status  | Response                  | Reason(s)                                                             |
+| ------- | ------------------------- | --------------------------------------------------------------------- |
+| [204][] | No Content                | Successfully removed the target resource's data retention policy.     |
+| [404][] | [JSON API error object][] | Target resource not found, or user unauthorized to perform action.    |
+
+### Request Body
+
+No request body.
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request DELETE \
+  https://app.terraform.io/api/v2/organizations/hashicorp/relationships/data-retention-policy
+```
+
+### Response
+
+No response body.
+
+Status code `204`.
+
```

Comment:
Non-blocking nitpick (same as above!): It looks like there's an extra line here ✂️ :
```suggestion
```